### ISSUE TITLE
all: Go 1.12.17 release status

### ISSUE THREAD

**Issue by toothrot** [golang/go#37199]
Issue tracking the go1.12.17 release by releasebot.

**Comment by toothrot** [comment]
A test failure occurred during the build:

```

##### GOMAXPROCS=2 runtime -cpu=1,2,4 -quick
--- FAIL: TestSpuriousWakeupsNeverHangSemasleep (12.94s)
    semasleep_test.go:77: Program failed to return on time and has to be killed, issue #27520 still exists

```

**Comment by toothrot** [comment]
## Log

    2020/02/12 12:56:40 starting
    2020/02/12 12:56:40 working in /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17
    2020/02/12 12:56:40 $ git fetch origin master
    2020/02/12 12:56:40 $ git fetch origin release-branch.go1.12
    2020/02/12 12:56:41 $ git clone --reference /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gitmirror -b release-branch.go1.12 https://go.googlesource.com/go /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gitwork
    2020/02/12 12:56:42 $ git codereview change relwork
    2020/02/12 12:56:43 $ git config gc.auto 0
    2020/02/12 12:56:44 $ git merge-base --is-ancestor deac3221fc4cd365fb40d269dd56551e9d354356 HEAD
    2020/02/12 12:56:44 $ git rev-parse go1.12.17
    2020/02/12 12:56:44 Release status issue title: "all: Go 1.12.17 release status"
    2020/02/12 12:56:44 Release status issue: https://golang.org/issue/37199
    2020/02/12 12:56:44 $ git add VERSION
    2020/02/12 12:56:44 $ git commit -m go1.12.17
    
    Change-Id: I9d398ed495011487544b1e5d0a469ae73c6f5927
     VERSION
    2020/02/12 12:56:45 $ git codereview mail -trybot
    2020/02/12 12:56:52 $ go get golang.org/x/build/cmd/release
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-ppc64le -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-386 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target freebsd-386 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target darwin-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target windows-386 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-armv6l -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-s390x -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target src -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target freebsd-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-arm64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 12:57:34 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target windows-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 13:13:24 release darwin-amd64:
    error from cmd/release binary = <nil>
    output from cmd/release binary:
    2020/02/12 12:57:34 darwin-amd64: Start.
    2020/02/12 12:57:34 darwin-amd64: Creating buildlet.
    2020/02/12 12:58:09 darwin-amd64: Pushing source to buildlet.
    2020/02/12 12:58:20 darwin-amd64: Writing VERSION file.
    2020/02/12 12:58:20 darwin-amd64: Cleaning goroot (pre-build).
    2020/02/12 12:58:20 darwin-amd64: Building (make.bash only).
    2020/02/12 13:00:51 darwin-amd64: Building race detector.
    2020/02/12 13:01:17 darwin-amd64: Building golang.org/x/tools/cmd/godoc.
    2020/02/12 13:01:25 darwin-amd64: Cleaning goroot (post-build).
    2020/02/12 13:01:26 darwin-amd64: Pushing and running releaselet.
    2020/02/12 13:01:29 darwin-amd64: Cleaning workdir.
    2020/02/12 13:01:30 darwin-amd64: Downloading tarball.
    2020/02/12 13:01:51 darwin-amd64: Wrote "/usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596/go1.12.17.darwin-amd64.tar.gz.untested".
    2020/02/12 13:01:51 darwin-amd64: Building (all.bash to ensure tests pass).
    2020/02/12 13:13:24 darwin-amd64: Error: Build failed: exit status 1
    Output:
    Building Go cmd/dist using /Users/gopher/goboot.
    Building Go toolchain1 using /Users/gopher/goboot.
    Building Go bootstrap cmd/go (go_bootstrap) using Go toolchain1.
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    Building Go toolchain2 using go_bootstrap and Go toolchain1.
    # runtime
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    # runtime
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    # runtime
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    # runtime
    go_asm.h:584:33: 'p' exponent requires hexadecimal mantissa
    go_asm.h:585:39: 'p' exponent requires hexadecimal mantissa
    Building Go toolchain3 using go_bootstrap and Go toolchain2.
    Building packages and commands for darwin/amd64.
    
    ##### Testing packages.
    ok  	archive/tar	0.479s
    ok  	archive/zip	2.556s
    ok  	bufio	0.265s
    ok  	bytes	4.158s
    ok  	compress/bzip2	0.293s
    ok  	compress/flate	1.656s
    ok  	compress/gzip	1.737s
    ok  	compress/lzw	0.258s
    ok  	compress/zlib	1.245s
    ok  	container/heap	0.138s
    ok  	container/list	0.149s
    ok  	container/ring	0.145s
    ok  	context	1.179s
    ok  	crypto	0.230s
    ok  	crypto/aes	0.177s
    ok  	crypto/cipher	0.238s
    ok  	crypto/des	0.131s
    ok  	crypto/dsa	0.201s
    ok  	crypto/ecdsa	0.449s
    ok  	crypto/elliptic	1.041s
    ok  	crypto/hmac	0.335s
    ok  	crypto/internal/subtle	0.121s
    ok  	crypto/md5	0.197s
    ok  	crypto/rand	0.189s
    ok  	crypto/rc4	0.336s
    ok  	crypto/rsa	0.307s
    ok  	crypto/sha1	0.244s
    ok  	crypto/sha256	0.180s
    ok  	crypto/sha512	0.147s
    ok  	crypto/subtle	0.142s
    ok  	crypto/tls	3.408s
    ok  	crypto/x509	3.984s
    ok  	database/sql	0.847s
    ok  	database/sql/driver	0.148s
    ok  	debug/dwarf	0.183s
    ok  	debug/elf	0.216s
    ok  	debug/gosym	5.127s
    ok  	debug/macho	0.134s
    ok  	debug/pe	0.184s
    ok  	debug/plan9obj	0.257s
    ok  	encoding/ascii85	0.163s
    ok  	encoding/asn1	0.348s
    ok  	encoding/base32	0.153s
    ok  	encoding/base64	0.224s
    ok  	encoding/binary	0.121s
    ok  	encoding/csv	0.136s
    ok  	encoding/gob	0.208s
    ok  	encoding/hex	0.144s
    ok  	encoding/json	0.975s
    ok  	encoding/pem	0.189s
    ok  	encoding/xml	0.222s
    ok  	errors	0.140s
    ok  	expvar	0.246s
    ok  	flag	0.165s
    ok  	fmt	0.299s
    ok  	go/ast	0.155s
    ok  	go/build	1.988s
    ok  	go/constant	0.242s
    ok  	go/doc	0.435s
    ok  	go/format	0.163s
    ok  	go/importer	0.680s
    ok  	go/internal/gccgoimporter	0.202s
    ok  	go/internal/gcimporter	4.551s
    ok  	go/internal/srcimporter	3.421s
    ok  	go/parser	0.213s
    ok  	go/printer	0.617s
    ok  	go/scanner	0.130s
    ok  	go/token	0.157s
    ok  	go/types	6.373s
    ok  	hash	0.167s
    ok  	hash/adler32	0.311s
    ok  	hash/crc32	0.140s
    ok  	hash/crc64	0.126s
    ok  	hash/fnv	0.121s
    ok  	html	0.156s
    ok  	html/template	0.175s
    ok  	image	0.381s
    ok  	image/color	0.166s
    ok  	image/draw	0.272s
    ok  	image/gif	0.711s
    ok  	image/jpeg	0.455s
    ok  	image/png	0.329s
    ok  	index/suffixarray	0.148s
    ok  	internal/cpu	0.131s
    ok  	internal/fmtsort	0.165s
    ok  	internal/poll	0.157s
    ok  	internal/singleflight	0.184s
    ok  	internal/trace	3.171s
    ok  	internal/x/crypto/chacha20poly1305	0.229s
    ok  	internal/x/crypto/cryptobyte	0.104s
    ok  	internal/x/crypto/curve25519	0.132s
    ok  	internal/x/crypto/hkdf	0.116s
    ok  	internal/x/crypto/internal/chacha20	0.215s
    ok  	internal/x/crypto/poly1305	0.113s
    ok  	internal/x/net/dns/dnsmessage	0.242s
    ok  	internal/x/net/http/httpguts	0.145s
    ok  	internal/x/net/http/httpproxy	0.113s
    ok  	internal/x/net/http2/hpack	0.105s
    ok  	internal/x/net/idna	0.119s
    ok  	internal/x/net/nettest	1.191s
    ok  	internal/x/net/route	0.133s
    ok  	internal/x/text/transform	0.126s
    ok  	internal/x/text/unicode/norm	0.104s
    ok  	internal/xcoff	0.187s
    ok  	io	0.173s
    ok  	io/ioutil	0.107s
    ok  	log	0.205s
    ok  	log/syslog	1.390s
    ok  	math	0.115s
    ok  	math/big	3.669s
    ok  	math/bits	0.130s
    ok  	math/cmplx	0.175s
    ok  	math/rand	0.828s
    ok  	mime	0.121s
    ok  	mime/multipart	0.910s
    ok  	mime/quotedprintable	0.338s
    ok  	net	25.847s
    ok  	net/http	5.692s
    ok  	net/http/cgi	1.697s
    ok  	net/http/cookiejar	0.215s
    ok  	net/http/fcgi	0.155s
    ok  	net/http/httptest	0.252s
    ok  	net/http/httptrace	0.154s
    ok  	net/http/httputil	0.492s
    ok  	net/http/internal	0.208s
    ok  	net/http/pprof	2.279s
    ok  	net/internal/socktest	0.257s
    ok  	net/mail	0.172s
    ok  	net/rpc	0.406s
    ok  	net/rpc/jsonrpc	0.680s
    ok  	net/smtp	0.513s
    ok  	net/textproto	0.541s
    ok  	net/url	0.131s
    ok  	os	1.560s
    ok  	os/exec	2.454s
    ok  	os/signal	5.075s
    ok  	os/user	0.123s
    ok  	path	0.113s
    ok  	path/filepath	0.243s
    ok  	plugin	0.244s
    ok  	reflect	0.998s
    ok  	regexp	0.298s
    ok  	regexp/syntax	1.589s
    ok  	runtime	65.489s
    ok  	runtime/debug	0.247s
    ok  	runtime/internal/atomic	0.261s
    ok  	runtime/internal/math	0.094s
    ok  	runtime/internal/sys	0.230s
    ok  	runtime/pprof	14.435s
    ok  	runtime/pprof/internal/profile	0.131s
    ok  	runtime/trace	6.033s
    ok  	sort	0.383s
    ok  	strconv	1.367s
    ok  	strings	4.848s
    ok  	sync	1.132s
    ok  	sync/atomic	0.155s
    ok  	syscall	0.251s
    ok  	testing	1.735s
    ok  	testing/quick	0.416s
    ok  	text/scanner	0.178s
    ok  	text/tabwriter	0.124s
    ok  	text/template	1.053s
    ok  	text/template/parse	0.128s
    ok  	time	3.019s
    ok  	unicode	0.186s
    ok  	unicode/utf16	0.122s
    ok  	unicode/utf8	0.162s
    ok  	cmd/addr2line	3.957s
    ok  	cmd/api	0.205s
    ok  	cmd/asm/internal/asm	1.311s
    ok  	cmd/asm/internal/lex	0.102s
    ok  	cmd/compile	0.833s
    ok  	cmd/compile/internal/gc	34.420s
    ok  	cmd/compile/internal/ssa	1.452s
    ok  	cmd/compile/internal/syntax	0.115s
    ok  	cmd/compile/internal/test	0.164s [no tests to run]
    ok  	cmd/compile/internal/types	0.114s
    ok  	cmd/cover	6.157s
    ok  	cmd/doc	0.300s
    ok  	cmd/fix	20.137s
    ok  	cmd/go	323.123s
    ok  	cmd/go/internal/cache	4.385s
    ok  	cmd/go/internal/dirhash	0.186s
    ok  	cmd/go/internal/generate	0.312s
    ok  	cmd/go/internal/get	0.392s
    ok  	cmd/go/internal/imports	0.108s
    ok  	cmd/go/internal/load	0.116s
    ok  	cmd/go/internal/lockedfile	0.102s
    ok  	cmd/go/internal/lockedfile/internal/filelock	0.133s
    ok  	cmd/go/internal/modconv	0.219s
    ok  	cmd/go/internal/modfetch	0.290s
    ok  	cmd/go/internal/modfetch/codehost	0.111s
    ok  	cmd/go/internal/modfile	0.116s
    ok  	cmd/go/internal/modload	0.695s
    ok  	cmd/go/internal/module	0.139s
    ok  	cmd/go/internal/mvs	0.487s
    ok  	cmd/go/internal/par	0.122s
    ok  	cmd/go/internal/search	0.095s
    ok  	cmd/go/internal/semver	0.083s
    ok  	cmd/go/internal/txtar	0.127s
    ok  	cmd/go/internal/web2	0.171s
    ok  	cmd/go/internal/work	0.151s
    ok  	cmd/gofmt	0.627s
    ok  	cmd/internal/buildid	0.922s
    ok  	cmd/internal/dwarf	0.131s
    ok  	cmd/internal/edit	0.137s
    ok  	cmd/internal/goobj	2.005s
    ok  	cmd/internal/obj	0.083s
    ok  	cmd/internal/obj/arm64	0.133s
    ok  	cmd/internal/obj/x86	1.738s
    ok  	cmd/internal/objabi	0.108s
    ok  	cmd/internal/src	0.652s
    ok  	cmd/internal/test2json	0.924s
    ok  	cmd/link	4.659s
    ok  	cmd/link/internal/ld	3.844s
    ok  	cmd/link/internal/sym	0.094s
    ok  	cmd/nm	7.438s
    ok  	cmd/objdump	9.465s
    ok  	cmd/pack	6.751s
    ok  	cmd/trace	0.444s
    ok  	cmd/vet	19.668s
    
    ##### os/user with tag osusergo
    ok  	os/user	0.141s
    
    ##### GOMAXPROCS=2 runtime -cpu=1,2,4 -quick
    --- FAIL: TestSpuriousWakeupsNeverHangSemasleep (12.94s)
        semasleep_test.go:77: Program failed to return on time and has to be killed, issue #27520 still exists
    FAIL
    FAIL	runtime	61.373s
    2020/02/12 13:13:22 Failed: exit status 1
    2020/02/12 13:13:24 FAILED
    2020/02/12 13:14:24 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target darwin-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 13:48:05 release darwin-amd64:
    error from cmd/release binary = <nil>
    output from cmd/release binary:
    2020/02/12 13:14:24 darwin-amd64: Start.
    2020/02/12 13:14:24 darwin-amd64: Creating buildlet.
    2020/02/12 13:14:36 darwin-amd64: Pushing source to buildlet.
    2020/02/12 13:14:50 darwin-amd64: Writing VERSION file.
    2020/02/12 13:14:50 darwin-amd64: Cleaning goroot (pre-build).
    2020/02/12 13:14:51 darwin-amd64: Building (make.bash only).
    2020/02/12 13:17:19 darwin-amd64: Building race detector.
    2020/02/12 13:17:47 darwin-amd64: Building golang.org/x/tools/cmd/godoc.
    2020/02/12 13:17:54 darwin-amd64: Cleaning goroot (post-build).
    2020/02/12 13:17:54 darwin-amd64: Pushing and running releaselet.
    2020/02/12 13:17:56 darwin-amd64: Cleaning workdir.
    2020/02/12 13:17:58 darwin-amd64: Downloading tarball.
    2020/02/12 13:18:18 user-go-dev-darwin-amd64-10_15-0: timeout after 10s waiting for headers for /status
    2020/02/12 13:18:38 user-go-dev-darwin-amd64-10_15-0: timeout after 10s waiting for headers for /status
    2020/02/12 13:18:57 Buildlet https://farmer.golang.org:443 failed three heartbeats; final error: 502 Bad Gateway
    2020/02/12 13:48:05 darwin-amd64: Wrote "/usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596/go1.12.17.darwin-amd64.tar.gz.untested".
    2020/02/12 13:48:05 darwin-amd64: Building (all.bash to ensure tests pass).
    2020/02/12 13:48:05 user-go-dev-darwin-amd64-10_15-0: peer dead with Buildlet https://farmer.golang.org:443 failed heartbeat after 9.001050779s; marking dead; err=502 Bad Gateway, waiting for headers for /exec
    2020/02/12 13:48:05 user-go-dev-darwin-amd64-10_15-0: peer dead with Buildlet https://farmer.golang.org:443 failed heartbeat after 9.001050779s; marking dead; err=502 Bad Gateway, waiting for headers for /halt
    2020/02/12 13:48:05 darwin-amd64: Error: Buildlet https://farmer.golang.org:443 failed heartbeat after 9.001050779s; marking dead; err=502 Bad Gateway
    2020/02/12 13:49:05 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target darwin-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/710ac70ab470e6ca61c53dfb453e6afc993b688f_549250596 -rev 710ac70ab470e6ca61c53dfb453e6afc993b688f
    2020/02/12 14:16:48 
    
    The release is ready.
    
    Please review and submit https://go-review.googlesource.com/q/I9d398ed495011487544b1e5d0a469ae73c6f5927
    and then run the release stage.

**Comment by toothrot** [comment]
## Latest build: go1\.12\.17

src ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.src.tar.gz))
linux\-386 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.linux-386.tar.gz))
linux\-armv6l ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.linux-armv6l.tar.gz))
linux\-amd64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.linux-amd64.tar.gz))
linux\-arm64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.linux-arm64.tar.gz))
freebsd\-386 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.freebsd-386.tar.gz))
freebsd\-amd64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.freebsd-amd64.tar.gz))
windows\-386 ([zip](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.windows-386.zip)) ([msi](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.windows-386.msi))
windows\-amd64 ([zip](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.windows-amd64.zip)) ([msi](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.windows-amd64.msi))
darwin\-amd64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.darwin-amd64.tar.gz))
linux\-s390x ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.linux-s390x.tar.gz))
linux\-ppc64le ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.12.17/go1.12.17.linux-ppc64le.tar.gz))

## Log

    2020/02/12 15:56:36 starting
    2020/02/12 15:56:36 working in /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17
    2020/02/12 15:56:36 $ git fetch origin master
    2020/02/12 15:56:37 $ git fetch origin release-branch.go1.12
    2020/02/12 15:56:38 $ git clone --reference /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gitmirror -b release-branch.go1.12 https://go.googlesource.com/go /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gitwork
    2020/02/12 15:56:40 $ git codereview change relwork
    2020/02/12 15:56:40 $ git config gc.auto 0
    2020/02/12 15:56:41 $ git merge-base --is-ancestor deac3221fc4cd365fb40d269dd56551e9d354356 HEAD
    2020/02/12 15:56:41 Release status issue title: "all: Go 1.12.17 release status"
    2020/02/12 15:56:41 Release status issue: https://golang.org/issue/37199
    2020/02/12 15:56:41 $ git rev-parse go1.12.17
    2020/02/12 15:57:11 $ git tag go1.12.17 46cb016190389b7e37b21f04e5343a628ca1f662
    2020/02/12 15:57:11 $ git push origin go1.12.17
    2020/02/12 15:57:17 $ go get golang.org/x/build/cmd/release
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-386 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target src -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-s390x -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-ppc64le -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target freebsd-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-arm64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target linux-armv6l -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target darwin-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target freebsd-386 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target windows-386 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 15:58:00 $ /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/gopath/bin/release -target windows-amd64 -user go-dev -version go1.12.17 -tools release-branch.go1.12 -net release-branch.go1.12 -staging_dir /usr/local/google/home/rakoczy/go-releasebot-work/go1.12.17/release-staging/46cb016190389b7e37b21f04e5343a628ca1f662_086853958 -rev 46cb016190389b7e37b21f04e5343a628ca1f662 -skip_tests
    2020/02/12 16:16:34 closing milestone Go1.12.17
    2020/02/12 16:16:34 
    
    The release run is complete! Refer to the playbook for the next steps.
    
    Thanks for riding with releasebot today.
